pkgs/kubectl-with-plugins/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv
+, fetchzip
+, sources
+, pkgs
+, makeWrapper
+}:
+with pkgs;
+rec {
+  plugins = {
```

Comment:
I would prefer to expose this in a `kubectl-plugins` attribute at the root of the toolbox

Reply:
default.nix
```
{
   ...

   kubectl-plugins = import ./pkgs/kubectl-plugins { ... };    # contains default.nix (attrset of plugins), script.nix

   kubectl-with-plugins = import ./pkgs/kubectl-with-plugins.nix 
     { plugins = kubectl-plugins, ... }; 
}
```